src/util/ivmFactory.js

```diff
@@ -243,6 +252,15 @@ async function createIvm(code, libraryVersionIds, versionId, secrets, testMode)
     }),
   );
 
+  await jail.set('_credential', function (key) {
+    if (_.isNil(credentials) || !_.isObject(credentials)) {
+      logger.error('Error fetching credentials map', versionId);
+      stats.increment('credential_error', { transformationId, workspaceId });
```

Comment:
I prefer stat as credential_errors. wdyt ?

Reply:
I think `credential_errors` is better, what about `credential_error_count`?

Reply:
it is count metric. Feels count is not needed